src/Microsoft.Graph.Core/Extensions/BaseRequestExtensions.cs

```diff
@@ -107,6 +107,29 @@ public static class BaseRequestExtensions
             return baseRequest;
         }
 
+        /// <summary>
+        /// Sets the maximum time for request retries to the default Retry Middleware Handler for this request.
+        /// This only works with the default Retry Middleware Handler.
+        /// If you use a custom Retry Middleware Handler, you have to handle it's retrieval in your implementation.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="baseRequest">The <see cref="BaseRequest"/> for the request.</param>
+        /// <param name="maxRetryTime">The maxRetryTime for the request.</param>
```

Comment:
Did you intend to rename this retriesTimeLimit? Please address throughout.

Reply:
Yes I do. Change duly effected. Thanks